models/stg_reddit_ads.yml

```diff
@@ -117,7 +117,7 @@ models:
       - name: account_id
         description: '{{ doc("account_id") }}'
       - name: ad_id
-        description: '{{ doc("ad_id") }}'
+        description: '{{ doc("ad_id") }}' 
```

Comment:
Should we also provide a not_null test for this field?

Reply:
good points, added not_null tests to each of these models, for their respective grain's field